proto/heartbeat.go

```diff
@@ -23,6 +23,13 @@ import (
 	"time"
 )
 
+// TODO(marc): we should assert
+// var _ security.AnyUserRequest = &PingRequest{}
+// here, but we need to break cycles first.
+
+// AnyUser implements security.AnyUserRequest.
+func (*PingRequest) AnyUser() {}
```

Comment:
We might want to use it to implement [`database/sql.(*DB).Ping`](http://godoc.org/database/sql#DB.Ping)